cross-chain/solana/programs/tbtc/src/lib.rs

```diff
@@ -0,0 +1,357 @@
+use anchor_lang::prelude::*;
+use anchor_spl::{
+    associated_token::AssociatedToken,
+    // mint,
+    token::{mint_to, TokenAccount, Mint as SplMint, MintTo as SplMintTo, Token as SplToken},
+};
+
+declare_id!("HksEtDgsXJV1BqcuhzbLRTmXp5gHgHJktieJCtQd3pG");
+
+#[program]
+pub mod tbtc {
+    use super::*;
+    
+    pub fn initialize(ctx: Context<Initialize>) -> Result<()> {
+        let tbtc = &mut ctx.accounts.tbtc; 
+        tbtc.authority = ctx.accounts.authority.key();
+        tbtc.token_mint = ctx.accounts.tbtc_mint.key();
+        tbtc.token_bump = *ctx.bumps.get("tbtc_mint").unwrap();
```

Comment:
You can just do `ctx.bumps["tbc_mint"]` because this key will exist for your account name.